BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1215,6 +1214,11 @@ IEnumerator FireReloadableMissile()
                 wpm.heatTarget = TargetSignatureData.noTarget;
             }
             ml.TargetPosition = transform.position + (multiLauncher ? vessel.ReferenceTransform.up * 5000 : transform.forward * 5000); //set initial target position so if no target update, missileBase will count a miss if it nears this point or is flying post-thrust
+            if (ml == null)
```

Comment:
This shouldn't be needed here with the check for it (and the gameObject) being `null` directly after the `yield return ...` that I added further up.